SYCL/Basic/buffer/buffer_dev_to_dev.cpp

```diff
@@ -21,21 +21,34 @@ using namespace cl::sycl;
 int main() {
   int Data[10] = {-1, -1, -1, -1, -1, -1, -1, -1, -1, -1};
   {
-    buffer<int, 1> Buffer(Data, range<1>(10),
-                          {property::buffer::use_host_ptr()});
 
-    default_selector Selector;
+    const std::vector<platform> Platforms = platform::get_platforms();
+
+    if (Platforms.size() < 2) {
```

Comment:
The test is executed with one of `*_RUN_PLACEHOLDER` set which might result in having only one platform here.
Probably, we should run the test without any `_RUN_PLACEHOLDER` value set.
@vladimirlaz What do you think?

Reply:
we have host device for all cases except acc AFAIR.
If host device does not fit test needs the test can be updated to run on OpenCL BE only